components/DualCurrencyInput.tsx

```diff
@@ -49,23 +50,27 @@ export function DualCurrencyInput({
       }
       setFiatAmount(text);
       if (getSatsAmount) {
-        setAmount(getSatsAmount(+text)?.toString() || "");
+        const numericValue = +text.replace(",", ".");
+        setAmount(getSatsAmount(numericValue)?.toString() || "");
       }
     }
   }
 
   function toggleInputMode() {
+    inputRef.current?.blur();
```

Comment:
minor: when converting back to fiat does it use the correct formatting?

Reply:
I tried to fix this using the `expo-localization` once for all, so that we can have the same type of decimal separator and digit grouping separators but there's an issue with Hermes not having good support for Intl in iOS which blocks this. I opened an [issue](https://github.com/getAlby/go/issues/305) to track this, but for now I'm merging as-is